sympy/printing/aesaracode.py

```diff
@@ -139,7 +139,7 @@ def _get_or_create(self, s, name=None, dtype=None, broadcastable=None):
         if key in self.cache:
             return self.cache[key]
 
-        value = aet.tensor(name=name, dtype=dtype, broadcastable=broadcastable)
+        value = aet.tensor(name=name, dtype=dtype, shape=broadcastable)
```

Comment:
This change is incompatible with Theano: https://github.com/sympy/sympy/pull/23745#discussion_r917436177

Reply:
I've reverted this change. We will need to consider separately how easy it will be to support both Theano and Aesara going forwards. At some point it definitely makes sense to just drop support for Theano.